packages/minifiers-css/urlrewriting-tests.js

```diff
@@ -32,12 +32,12 @@ Tinytest.add("minifiers - url rewriting when merging", function (test) {
   t('http://', 'http://', 'malformed URL');
 
   parseOptions.source = 'application/client/dir/other-style.css';
-  t('./image.png', '/image.png', 'base path is root');
-  t('./child/image.png', '/child/image.png', 'child directory from root');
-  t('child/image.png', '/child/image.png', 'child directory from root');
-  t('/image.png', '/image.png', 'absolute url');
-  t('"/image.png"', '"/image.png"', 'double quoted url');
-  t("'/image.png'", "'/image.png'", 'single quoted url');
+  t('./image.png', 'image.png', 'base path is root');
```

Comment:
can you please explain again on this concrete example why `/image.png` is incorrect and how `image.png` is correct if the ROOT_URL is set to `http://site.com/subpath/`?